Packs/RiskSense/TestPlaybooks/playbook-RiskSenseApplyTag-Test.yml

```diff
@@ -0,0 +1,270 @@
+id: RiskSense Apply Tag - Test
+version: -1
+name: RiskSense Apply Tag - Test
```

Comment:
@anara123 do you want us to add `fromversion: 5.0.0` here ?

Reply:
yes please, add here too

Reply:
Got it